### ISSUE TITLE
Feature: open on current file or current working directory

### ISSUE THREAD

**Issue by alextes** [cocopon/vaffle.vim#52]
TL;DR: have vaffle open the directory for the current buffer and if the current buffer is empty the current working directory.

From dirvish I'm used to having one button to jump into file-oriented action. I bound this to opening Vaffle for the current buffer file because that's the scenario I go through most of the time. Moving files, renaming, creating, etc. However, I also open nvim a lot starting with an empty buffer, or sometimes I clear all my buffers to clean up my session, in both scenarios, I can't use my one keybinding anymore as '%' is not associated with a file. dirvish handled these scenarios seamlessly I realize now by assuming that if there is no file you probably want to open in the working dir. Could this be desirable behavior for vaffle too?

I'm strongly preferring how Vaffle deals with actions on files, but this and #49 are keeping me from switching currently.

**Comment by cocopon** [comment]
> TL;DR: have vaffle open the directory for the current buffer and if the current buffer is empty the current working directory.

Do you mean you want to open Vaffle with...
- the current directory for an empty buffer 
- or the parent directory for an opened file?

If so, you can do it with the following command:

```vim
:call vaffle#init(expand('%'))
```

Here is my [.vimrc](https://github.com/cocopon/dotfiles/blob/24322c6089072c9b0f5a230bd0dc8f185236e076/.vimrc#L272-L273) settings for example:
```vim
" Open the parent directory, or the current directory if empty
nnoremap <silent> <C-u> :<C-u>call vaffle#init(expand('%'))<CR>
```

I think the current working directory should be respected by default.

**Comment by alextes** [comment]
That's exactly what I want. Thanks!
